src/databricks/labs/dqx/installer/config_provider.py

```diff
@@ -17,12 +17,18 @@ def __init__(self, prompts: Prompts, warehouse_configurator: WarehouseInstaller)
         self._prompts = prompts
         self._warehouse_configurator = warehouse_configurator
 
-    def prompt_new_installation(self) -> WorkspaceConfig:
+    def prompt_new_installation(self, install_folder: str | None = None) -> WorkspaceConfig:
         logger.info(
             "Please answer a couple of questions to provide default DQX run configuration. "
             "The configuration can also be updated manually after the installation."
         )
 
+        # Show installation folder information
+        if install_folder:
+            logger.info(f"DQX will be installed in folder '{install_folder}'")
+        else:
+            logger.info("DQX will be installed in the default location (user home or global based on environment)")
```

Comment:
we could say whether this will be a user home or global installation as we have access to the env vars

Reply:
updated